src/pages/Login/style.scss

```diff
@@ -0,0 +1,17 @@
+.card-selected {
+  border: 2.5px solid #8F74DD;
+}
+
+.content-login-padding {
+  margin: 0 6em 2em !important;
+}
+
+.card-span {
+  font-weight: bold;
+  margin-top: 1.5em;
+  margin-bottom: 24px;
+}
+
+.ant-card-body {
```

Comment:
É bom unir todas as alterações do ant design naquele arquivo na raiz do projeto. Também é bom evitar alterar a classe como um todo, se for algo pontual, cria uma nova classe e coloca no elemento para evitar de fazer alteraçōes globais

Reply:
Se conseguir, aplica o BEM no css https://medium.com/trainingcenter/bem-em-5min-f5c80fd23439
Toda classe eh global, mesmo no arquivo da pasta login. Para evitar do css vazar, você pode fazer algo como:

```
.screen-login {
  .card-selected {
    border: 2.5px solid #8F74DD;
  }
  
  .content-login-padding {
    margin: 0 6em 2em !important;
  }
  
  .card-span {
    font-weight: bold;
    margin-top: 1.5em;
    margin-bottom: 24px;
  }
}
```

e a classe screen-login fica na div mais externa da tela de login